ui/src/pages/project-details/project-details-form.tsx

```diff
@@ -0,0 +1,120 @@
+import { FormController } from 'components/form/form-controller'
```

Comment:
@mihow, here is the form, using all the helpers from previous PR:s. I have tried to find a good balance between flexibility and simplicity, by using smaller building blocks rather than one massive super generic form component. I'm a bit inspired by the Radix UI components (checkout the code example): https://www.radix-ui.com/primitives/docs/components/accordion